runtime/sunvmi/sunvmi.c

```diff
@@ -217,7 +217,29 @@ getCallerClassJEP176Iterator(J9VMThread * currentThread, J9StackWalkState * walk
 
 	Assert_SunVMI_mustHaveVMAccess(currentThread);
 
-	if (J9_ARE_ALL_BITS_SET(J9_ROM_METHOD_FROM_RAM_METHOD(walkState->method)->modifiers, J9AccMethodFrameIteratorSkip)) {
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE) && (JAVA_SPEC_VERSION == 11)
```

Comment:
Do these changes also apply to Java 8?
```suggestion
#if defined(J9VM_OPT_OPENJDK_METHODHANDLE) && (JAVA_SPEC_VERSION <= 11)
```

Reply:
I'd assume so but hadn't tested yet.